.github/workflows/dev-testing.yml

```
@@ -48,6 +48,8 @@ jobs:
 
       - name: Dev gallery - ROIExtractors
         run: pip install git+https://github.com/CatalystNeuro/roiextractors@main
+      - name: Dev gallery - DANDI
+        run: pip install git+https://github.com/dandi/dandi-cli@master
```

Looking at the print out that you added this seems to be working. But you could specify the -no-cache-dir version to avoid nasty surprises.

```suggestion
  run: pip install --no-cache-dir git+https://github.com/dandi/dandi-cli@master
```

---

added to all